models/quality_measures/intermediate/cqm130_documentation_of_current_medications_in_the_medical_record/quality_measures__int_cqm130_denominator.sql

```diff
@@ -0,0 +1,191 @@
+    from {{ ref('quality_measures__stg_core__encounter') }} encounter
+    inner join {{ ref('quality_measures__int_cqm130__performance_period') }} as pp
+        on coalesce(encounter.encounter_end_date,encounter.encounter_start_date) >= pp.performance_period_begin
+            and coalesce(encounter.encounter_start_date,encounter.encounter_end_date) <= pp.performance_period_end
+    where lower(encounter_type) in (
+          'home health'
+        , 'office visit'
+        , 'outpatient'
+        , 'outpatient rehabilitation'
+        , 'telehealth'
+    )
+
+)
+
+, procedure_encounters as (
+
+    select 
+          patient_id
+        , procedure_date as procedure_encounter_date
+        , {{ try_to_cast_date('null', 'YYYY-MM-DD') }} as claims_encounter_date
+    from {{ ref('quality_measures__stg_core__procedure') }} procs
+    inner join {{ ref('quality_measures__int_cqm130__performance_period') }} as pp
+        on procedure_date between pp.performance_period_begin and pp.performance_period_end
+    inner join visit_codes
+        on coalesce(procs.normalized_code,procs.source_code) = visit_codes.code
+
+)
+
+, claims_encounters as (
+    
+    select 
+          patient_id
+        , {{ try_to_cast_date('null', 'YYYY-MM-DD') }} as procedure_encounter_date
+        , coalesce(claim_end_date,claim_start_date) as claims_encounter_date
+    from {{ ref('quality_measures__stg_medical_claim') }} medical_claim
+    inner join {{ ref('quality_measures__int_cqm130__performance_period') }} as pp 
+        on coalesce(claim_end_date,claim_start_date)  >=  pp.performance_period_begin
+            and coalesce(claim_start_date,claim_end_date) <=  pp.performance_period_end
+    inner join visit_codes
+        on medical_claim.hcpcs_code = visit_codes.code
+
+)
+
+, all_encounters as (
+
+    select *, 'v' as visit_enc, cast(null as {{ dbt.type_string() }}) as proc_enc, cast(null as {{ dbt.type_string() }}) as claim_enc
+    from visits_encounters
+
+    union all
+
+    select *, cast(null as {{ dbt.type_string() }}) as visit_enc, 'p' as proc_enc, cast(null as {{ dbt.type_string() }}) as claim_enc
+    from procedure_encounters
+
+    union all
+    
+    select *, cast(null as {{ dbt.type_string() }}) as visit_enc, cast(null as {{ dbt.type_string() }}) as proc_enc, 'c' as claim_enc
+    from claims_encounters
+
+)
+
+, multiple_encounters_by_patient as (
+
+    select
+          patient_id
+        , procedure_encounter_date
+        , claims_encounter_date
+        , case when procedure_encounter_date >= claims_encounter_date
+                then procedure_encounter_date
+            else claims_encounter_date
+          end as max_encounter_date
+        , concat(concat(
+              coalesce(min(visit_enc),'')
+            , coalesce(min(proc_enc),''))
+            , coalesce(min(claim_enc),'')
+            ) as qualifying_types
```

Comment:
We are in the middle of adding support for other data warehouses. Can you make sure to use the dbt concat function instead? 

`{{ dbt.concat(["column_1", "column_2"]) }}`

Reply:
This is the only change required. The CI tests passed for the other data warehouses. Thanks, @utsavpaudel!